code/modules/projectiles/guns/projectile/shotgun.dm

```diff
@@ -255,3 +255,21 @@
 		toggle_folded(user)
 		return FALSE
 	return ..()
+
+/obj/item/gun/projectile/shotgun/foldable/cameragun
+	name = "camera"
+	desc = "A polaroid camera"
+	desc_extended = "Upon closer inspection, this is not a camera at all, but a 9mm firearm concealed inside the shell of one, which can be deployed by pressing a button."
```

Comment:
As fluffy mentioned above, that's going to have to be changed.

Reply:
Done